src/Akka.Hosting.TestKit/Akka.Hosting.TestKit.csproj

```diff
@@ -5,6 +5,7 @@
         <TargetFramework>$(LibraryFramework)</TargetFramework>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <LangVersion>8.0</LangVersion>
+        <IsPackable>true</IsPackable>
```

Comment:
Seems like a quirk in NUKE. For this project, IsPackable needs to be set explicitly for it to be packed.